comms-commands/orb.yml

```diff
@@ -19,8 +19,8 @@ jobs:
     steps:
       - terraform_plan:
           path: << parameters.path >>
-          backend_config_file: ./environments/<< parameters.environment >>/<< parameters.environment >>.backend
-          var_file: ./environments/<< parameters.environment >>/<< parameters.environment >>.tfvars
+          backend_config_file: ./backends/<< parameters.environment >>.backend
```

Comment:
wouldn't this still force you to duplicate the backend files for each module?

Reply:
Good point. Ok I think `backend_config_file` and `var_file` are both relative to `path`. So how about this:

`backend_config_file: ../terraform-backends/<< parameters.environment >>.backend`

Then in a single module project we just have:

```
terraform
├── variables
│   ├── prd.tfvars
│   └── uat.tfvars
└── main.tf
terraform-backends
├── prd.backend
└── uat.backend
```

And in a multiple module project we have:

```


terraform
├── module-A
│   ├── variables
│   │   ├── prd.tfvars
│   │   └── uat.tfvars
│   └── main.tf
├── module-B
│   ├── variables
│   │   ├── prd.tfvars
│   │   └── uat.tfvars
│   └── main.tf
└── terraform-backends
    ├── prd.backend
    └── uat.backend

```

Reply:
Yeah I'm ok with that. I'd try to keep it simple and only use structure n. 2.
We can treat single module projects as multi-module with just 1 module.

I don't think that would make a difference for this PR anyway, just mentioning.